docs/esp32/quickref.rst

```diff
@@ -102,6 +102,32 @@ Once the network is established the :mod:`socket <usocket>` module can be used
 to create and use TCP/UDP sockets as usual, and the ``urequests`` module for
 convenient HTTP requests.
 
+LAN
+^^^
+
+To use the wired interfaces one has to specify the pins ::
+
+    import network
+
+    lan = network.LAN(mdc=PIN_MDC, ...)   # Set the pin configuration
+    lan.active(True)                      # activate the interface
+    lan.ifconfig()                        # get the interface's IP/netmask/gw/DNS addresses
+
+
+Pinouts for popular boards
+^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
this heading is not needed, it's enough to just make this section a paragraph (to keep the quickref short and to the point)